APE14.rst

```diff
@@ -406,6 +426,74 @@ dimension encoding time-of-observation.
     wcs.world_axis_object_components = [('spec', 0, 'value')]
     wcs.world_axis_object_classes  = {'spec':('astropy.units.pixel': {})}
 
+Pixel and world coordinate ordering
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The API above provides a way to distinguish between pixel coordinates defined
+using the standard Cartesian ordering (x, y) and array indices defined using the
+(row, column) ordering. For example, values returned from
+``world_to_pixel_values`` would be in the correct order to use for plotting
+using e.g. Matplotlib, while values returned from ``world_to_index_values``
+would be in the correct order to use for indexing a Numpy array. Both are valid
```

Comment:
``Both are valid`` -> ``Both are valid in different contexts`` ?

Reply:
Agree

Reply:
Fixed in 0b34b6ccb10618c2ebdb2c24e416d2df0bb5014f